### ISSUE TITLE
log flood

### ISSUE THREAD

**Issue by systemcrash** [happyDomain/happydomain#7]
When a domain is imported, and I try to manage it, the logs flood with e.g.
```
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.163009ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.321826ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.150704ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |      2.2032ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.382012ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.134506ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.139739ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.136728ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.128112ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.131328ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.145964ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.127985ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.127622ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.123439ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.140447ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.134038ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.177467ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.131711ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.188089ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.150395ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.127513ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.197477ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.409186ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.504922ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.161165ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |     4.49549ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.265191ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.194975ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.235892ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.197206ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |     2.30671ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.217499ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |    2.331856ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
[GIN] 2023/02/23 - 02:30:56 | 200 |     2.21467ms |      192.0.2.1 | GET      "/api/domains/mxhAl5GPc0u08OCZ8H-v_g"
```

Is this normal?

**Comment by nemunaire** [comment]
Fixed in 7536023d62e22bd8e4d8034cbf75c2c5b105a24c.

It appears recently with a module update. I don't understand the root cause of this. But excluding sveltestrap here stops the reactive change that generates the flood.

**Comment by systemcrash** [comment]
> Fixed in [7536023](https://github.com/happyDomain/happydomain/commit/7536023d62e22bd8e4d8034cbf75c2c5b105a24c).
> 

Confirmed.
